src/data/sample-ingredient-data.js

```diff
@@ -0,0 +1,109 @@
+let sampleIngredientData = [
+  {
+    "id": 20081,
+    "name": "wheat flour",
+    "estimatedCostInCents": 142
+  },
+  {
+    "id": 18372,
+    "name": "bicarbonate of soda",
+    "estimatedCostInCents": 582
+  },
+  {
+    "id": 1123,
+    "name": "eggs",
+    "estimatedCostInCents": 472
+  },
+  {
+    "id": 19335,
+    "name": "sucrose",
+    "estimatedCostInCents": 902
+  },
+  {
+    "id": 19206,
+    "name": "instant vanilla pudding",
+    "estimatedCostInCents": 660
+  },
+  {
+    "id": 19334,
+    "name": "brown sugar",
+    "estimatedCostInCents": 559
+  },
+  {
+    "id": 2047,
+    "name": "salt",
+    "estimatedCostInCents": 280
+  },
+  {
+    "id": 1012047,
+    "name": "fine sea salt",
+    "estimatedCostInCents": 528
+  },
+  {
+    "id": 10019903,
+    "name": "semi sweet chips",
+    "estimatedCostInCents": 253
+  },
+  {
+    "id": 1145,
+    "name": "unsalted butter",
+    "estimatedCostInCents": 617
+  },
+  {
+    "id": 2050,
+    "name": "vanilla",
+    "estimatedCostInCents": 926
+  },
+  {
+    "id": 20027,
+    "name": "corn starch",
+    "estimatedCostInCents": 236
+  },
+  {
+    "id": 1002046,
+    "name": "dijon style mustard",
+    "estimatedCostInCents": 619
+  },
+  {
+    "id": 11215,
+    "name": "whole garlic clove",
+    "estimatedCostInCents": 220
+  },
+  {
+    "id": 1012046,
+    "name": "whole grain dijon mustard",
+    "estimatedCostInCents": 867
+  },
+  {
+    "id": 19911,
+    "name": "maple",
+    "estimatedCostInCents": 349
+  },
+  {
+    "id": 16112,
+    "name": "miso",
+    "estimatedCostInCents": 978
+  },
+  {
+    "id": 10010062,
+    "name": "pork chop",
+    "estimatedCostInCents": 834
+  },
+  {
+    "id": 1102047,
+    "name": "s&p",
+    "estimatedCostInCents": 524
+  },
+  {
+    "id": 16124,
+    "name": "soy sauce",
+    "estimatedCostInCents": 486
+  },
+  {
+    "id": 1016168,
+    "name": "sriracha sauce",
+    "estimatedCostInCents": 576
+  },
+]
+
+export default sampleIngredientData;
```

Comment:
I see that we are doing a test file for the information which seems like a good idea so that we don't have to use all the user's data